packages/components/src/components/Card/Card.tsx

```diff
@@ -1,16 +1,24 @@
 import { forwardRef, ReactNode } from 'react';
 import styled from 'styled-components';
-import { borders, spacings } from '@trezor/theme';
+import { borders, mapElevationToBoxShadow, spacings } from '@trezor/theme';
+import { ElevationContext, useElevation } from '../ElevationContext/ElevationContext';
+import { Elevation, mapElevationToBackground } from '@trezor/theme/src/elevation';
 
 const Wrapper = styled.div<{ $elevation: Elevation; $paddingSize: number }>`
     display: flex;
     flex-direction: column;
     padding: ${({ $paddingSize }) => $paddingSize}px;
-    background: ${({ theme }) => theme.backgroundSurfaceElevation1};
+    background: ${({ theme, $elevation }) => theme[mapElevationToBackground[$elevation]]};
```

Comment:
How about creating this pluggable thingy to not repeat the same code?

```
const backgroundElevation = css<{ $elevation: Elevation }>`
    background: ${({ theme, $elevation }) => theme[mapElevationToBackground[$elevation]]};
`;
```

and then do this whenever needed:

```suggestion
    background: ${backgroundElevation};
```

Reply:
Thats good idea!

but where to put it? `theme` would be good but it has no dependency on styled components

Reply:
agrred to not include this